library/std/src/sys/windows/ext/process.rs

```diff
@@ -106,4 +119,104 @@ impl CommandExt for process::Command {
         self.as_inner_mut().creation_flags(flags);
         self
     }
+
+    fn arg_ext(&mut self, arg: impl Arg) -> &mut process::Command {
+        self.as_inner_mut().arg_ext(arg);
+        self
+    }
+
+    fn args_ext(&mut self, args: impl IntoIterator<Item = impl Arg>) -> &mut process::Command {
+        for arg in args {
+            self.arg_ext(arg);
+        }
+        self
+    }
+}
+
+/// Traits for handling a sized command.
+// FIXME: This really should be somewhere else, since it will be duplicated for unix. sys_common? I have no idea.
+// The implementations should apply to unix, but describing it to the type system is another thing.
+#[unstable(feature = "command_sized", issue = "74549")]
+pub trait CommandSized: core::marker::Sized {
+    /// Possibly pass an argument.
+    fn maybe_arg(&mut self, arg: impl Arg) -> io::Result<&mut Self>;
+    /// Possibly pass many arguments.
```

Comment:
```suggestion
    /// Possibly pass many arguments. Returns an error if the size of the arguments would overflow the command line. The error contains the number of arguments added as well as the reason the remaining arguments could not be added.
```
Not sure if my comment is correct, but the error conditions should be documented, so types implementing the trait know what to do.